packages/components/src/templates/next/components/internal/Footer/Footer.tsx

```diff
@@ -379,7 +379,7 @@ const FooterDesktop = ({
   siteMapLink,
```

Comment:
![Screenshot 2024-09-23 at 9.10.56 PM.png](https://graphite-user-uploaded-assets-prod.s3.amazonaws.com/AWVfQY5lNPJPEvYNU3KB/f869f48d-848a-4017-ae65-83f39eee6570.png)

This vertical gap between the site name and the botton seems to be quite large

Reply:
Or is it because we didn't add footer items in storybook?

Reply:
Yeah, but will fix this!

Reply:
@harishv7 This wasn't fixable on my end, the div itself is created even if there are no navigation items so the padding still appears. It looks like we can't create a footer without navigation items so for now we'll assume this doesn't happen, need engineer's help later on for cases where the site's navbar is empty (not priority now?)

But I adjusted some other gaps like the one between the site name and the 1st nav item in https://github.com/opengovsg/isomer/pull/676/commits/e866254fc35cfe8c80e0f0a0bea2dc937bc01582